app/resources/forms.py

```diff
@@ -0,0 +1,39 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    IntegerField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    autocomplete = StringField('Enter the address')
+
+    name = StringField('Name', validators=[
+        InputRequired(),
+        Length(1, 64)
+    ])
+    description = StringField('Description', validators=[
+        InputRequired(),
+    ])
+    website = StringField('Website')
+
+    street_number = IntegerField('Street Number', validators=[
+        InputRequired()
+    ])
+    route = StringField('Street Address', validators=[
+        InputRequired()
+    ])
+    locality = StringField('City', validators=[
+        InputRequired()
+    ])
+    administrative_area_level_1 = StringField('State', validators=[
+        InputRequired()
+    ])
+    postal_code = StringField('ZIP Code', validators=[
+        InputRequired(),
+        Length(5)
```

Comment:
does google generate a normal zip code or a zip+4? We should try and make sure we are only getting 5 digit zips as our uniqueness depends on that. If that's going to be a problem, we might want to add some code to strip out the first five digits in the ZIPCode constructor.